content/topics/plain-language/_index.md

```diff
@@ -33,7 +33,7 @@ data:
   communities:
     multiple: false
     community:
-      - link: "https://digital.gov/communities/plain-language/"
+      - link: "/communities/plain-language-community-of-practice"
```

Comment:
Some notes on this change:

1. Using a relative url (standard convention). 
2. The featured resource component requires a resource, not an alias. Because of this we need to specify the exact name of the resource instead of of the alias shorthand.